### PR TITLE
kwalletcli: fix kwalletcli_getpin command

### PR DESCRIPTION
**Pull request by niknetniko**
###### Description of changes

Fix the command `kwalletcli_getpin` from `kwalletcli`, in the same way `pinentry-kwallet` was fixed.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by peterhoeg** [comment]
I'm not using this (`kwallet-query` from `kwalletd` works for me). How would one verify that this works?

**Comment by niknetniko** [comment]
> I'm not using this (`kwallet-query` from `kwalletd` works for me). How would one verify that this works?

If you install `kwalletcli` you should be able to run the `kwalletcli_getpin` command, but I assume you need Plasma (or at least kwallet) for this, so I'm not totally sure (I am running plasma, so it wasn't issue for me).

File: pkgs/tools/security/kwalletcli/default.nix [hunk 1]
```diff
@@ -38,6 +41,9 @@ mkDerivation rec {
     wrapProgram $out/bin/pinentry-kwallet \
       --prefix PATH : $out/bin:${lib.makeBinPath [ pinentry-qt ]} \
       --set-default PINENTRY pinentry-qt
+    wrapProgram $out/bin/kwalletcli_getpin \
+      --prefix PATH : $out/bin:${lib.makeBinPath [ pinentry-qt ]} \
+      --set-default PINENTRY pinentry-qt
```
Comment (SuperSandro2000) [inline pkgs/tools/security/kwalletcli/default.nix:46]:
```suggestion
    for program in pinentry-kwallet kwalletcli_getpin; do
      wrapProgram $out/bin/$program \
        --prefix PATH : $out/bin:${lib.makeBinPath [ pinentry-qt ]} \
        --set-default PINENTRY pinentry-qt
    done
```

**Comment by SuperSandro2000** [comment]
Tested locally and this fixed executing kwalletcli_getpin

**Pull request merged**